plugins/snmp_receiver_plugin/mibs/standard/BRIDGE-MIB.mib

```diff
@@ -0,0 +1,1085 @@
+          BRIDGE-MIB DEFINITIONS ::= BEGIN
```

Comment:
check with Meir if required for IB

Reply:
There are some dependencies inside some of them, so let's keep all of them for now